tests/functional/tests/bundles/base-yoga-ovn.yaml

```diff
@@ -134,10 +134,18 @@ applications:
     charm: ch:nova-compute
     channel: yoga/stable
     num_units: 1
-    storage:
-      ephemeral-device: '40G'
     options:
+      enable-live-migration: true
+      enable-resize: true
+      encrypt: true
+      force-raw-images: true
+      libvirt-image-backend: rbd
+      migration-auth-type: ssh
+      virt-type: lxc
       openstack-origin: *openstack-origin
+      verbose: true
+      debug: true
+    constraints: arch=amd64 mem=4096
```

Comment:
constraints should be added to the machines themselves (eg. for this unit, look for constraints on machine `9`)

Also `arch=amd64` is in the default constraints, and I believe mem and cpu constraints don't have any effect for containers. (although here we'll want to use a VM for nova-compute anyway, where mem and cpu constraints do take effect)